internal/pkg/template/templates/workloads/services/rd-web/cf.yml

```diff
@@ -101,7 +101,12 @@ Resources:
               - Name: {{toSnakeCase $var}}
                 Value:
                   Fn::GetAtt: [ {{$stackName}}, Outputs.{{$var}}]
-              {{- end}}
+              {{- end }}
+              {{- range $var := .NestedStack.SecretOutputs }}
+              - Name: {{toSnakeCase $var}}_ARN
+                Value:
+                  Fn::GetAtt: [ {{$stackName}}, Outputs.{{$var}}]
+              {{- end }}
```

Comment:
I added secrets arns as environment variables because users have to fetch the value of the secret from their application code. Therefore, they need to know the ARN.

In order to be **future compatible** for when app runner supports secrets, we're writing these env var names with the `_ARN` suffix. If we did not do this and the service starts supporting secrets then the environment variable names will collide.